From 0b5906437f3188443a88daa926410256a9e16e24 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Sun, 17 Jul 2016 15:39:49 +0300
Subject: [PATCH 0351/1345] dts: a80x0: enable RGMII port on CP1

commit  38d435a9212371ab955fedc45b88a5f6c8ac1b71 from
https://github.com/MarvellEmbeddedProcessors/linux-marvell.git

- enable RGMII port on CP1 GOP2
- disable unused GOP3 port
- patch disable unconnected SGMII port
  and enable supported RGMII port

Change-Id: If08fed7acb72b3bffcf209703a0117349257dea3
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31175
Reviewed-by: Hanna Hawa <hannah@marvell.com>
Tested-by: Hanna Hawa <hannah@marvell.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/marvell/armada-8040-db-router.dts     |   13 ++++++-------
 1 file changed, 6 insertions(+), 7 deletions(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
index 2d46cda..c7df842 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-db-router.dts
@@ -126,14 +126,13 @@
 					status = "okay";
 				};
 				emac2_1: mac2 {
-					phy-mode = "rgmii"; /* lane-0 */
+					phy-mode = "rgmii"; /* rgmii-0 */
 					phy = <&phy2>;
-					status = "disabled";
+					status = "okay";
 				};
 				emac3_1: mac3 {
-					phy-mode = "rgmii"; /* rgmii-1 */
-					phy = <&phy3>;
-					status = "okay";
+					phy-mode = "sgmii"; /* lane-4 */
+					status = "disabled";
 				};
 			};
 			ppv22@000000 {
@@ -141,10 +140,10 @@
 					status = "okay";
 				};
 				eth1_1: eth1@020000 {
-					status = "disabled";
+					status = "okay";
 				};
 				eth2_1: eth2@030000 {
-					status = "okay";
+					status = "disabled";
 				};
 			};
 		};
-- 
1.7.9.5

